ChatAppGenAI/ModelRunner.cs

```diff
@@ -13,9 +13,12 @@
 
 namespace ChatAppGenAI
 {
-    public class Phi3Runner : IDisposable
+    public class ModelRunner : IDisposable
     {
-        private readonly string ModelDir = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "phi3");
+        // huggingface-cli download microsoft/Phi-3-mini-4k-instruct-onnx --include directml/* --local-dir .
+        private readonly string ModelDir = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, @"Models\directml\directml-int4-awq-block-128");
```

Comment:
If we are changing the folder name for the model, the readme should also be updated